iothub/device/src/Transport/Mqtt/IWillMessage.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.Azure.Devices.Client.Transport.Mqtt
+namespace Microsoft.Azure.Devices.Client
```

Comment:
The amqp public types are under the "root" namespace, so moving this there as well